### PR TITLE
Add build script to remove x86_64 and i386 archs from frameworks

### PR DESCRIPTION
**Pull request by josep1972**
Added a Run Script step in Build Phases of the Loop target that removes simulator architectures

### CODE REVIEW

File: Loop.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2445,6 +2449,23 @@
...
+			shellPath = /bin/sh;
+			shellScript = " #!/usr/bin/env bash\n\nAPP_PATH=\"${TARGET_BUILD_DIR}/${WRAPPER_NAME}\"\n\n# This script loops through the frameworks embedded in the application and\n# removes x86_64 and i386 architectures.\nfind \"$APP_PATH\" -name '*.framework' -type d | while read -r FRAMEWORK\ndo\n    FRAMEWORK_EXECUTABLE_NAME=$(defaults read \"$FRAMEWORK/Info.plist\" CFBundleExecutable)\n    FRAMEWORK_EXECUTABLE_PATH=\"$FRAMEWORK/$FRAMEWORK_EXECUTABLE_NAME\"\n\n    if [ ! -f \"${FRAMEWORK_EXECUTABLE_PATH}\" ]; then\n        continue\n    fi\n\n    echo \"Thinning framework $FRAMEWORK_EXECUTABLE_NAME\"\n    \n    if xcrun lipo -info \"${FRAMEWORK_EXECUTABLE_PATH}\" | grep --silent \"Non-fat\"; then\n        echo \"   Framework non-fat, skipping\"\n        continue\n    fi\n\n    ARCHS=$(lipo -archs \"$FRAMEWORK_EXECUTABLE_PATH\")\n\n    for ARCH in $ARCHS\n    do\n       if [ \"$ARCH\" == \"x86_64\" ] || [ \"$ARCH\" == \"i386\" ]; then\n        echo \"   Removing $ARCH from $FRAMEWORK_EXECUTABLE_NAME\"\n        xcrun lipo -remove \"$ARCH\" \"$FRAMEWORK_EXECUTABLE_PATH\" -o \"$FRAMEWORK_EXECUTABLE_PATH\"\n       fi\n    done\ndone\n";
```
Comment (ps2) [inline Loop.xcodeproj/project.pbxproj]: Please split this out into a separate script.

Comment (josep1972) [reply Loop.xcodeproj/project.pbxproj]: New script created.

**Comment by ps2** [comment]
Can you describe the issue that this fixes, and steps to test?

**Comment by josep1972** [comment]
This update addresses a failure when creating IPA for distribution. 
There are two ways to test if the issue has been addressed: 
1) Archive and then Validate App or 
2) Archive and then create app for Ad-Hoc distribution.

**Comment by shahzadpandher** [comment]
I had the same issue but I was able to export it as ipa as said above

**Comment by josep1972** [comment]
Any word on adding this?

**Comment by josep1972** [comment]
merge level with LoopKit dev to fix conflicts

**Comment by josep1972** [comment]
Close due to bad merge

**Pull request closed without merging**